steps/dexprotector/1.0.1/step.yml

```diff
@@ -0,0 +1,191 @@
+published_at: 2021-04-26T18:42:25.769554+08:00
+source:
+  git: https://github.com/dexprotector/bitrise-step-dexprotector.git
+  commit: 73ee52a2f25791aa6fe43e85f392c2ee08402943
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- ios
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: java
+  apt_get:
+  - name: java
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- app_file: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: 'Path to the app file to be protected. For example: `$BITRISE_APK_PATH`
+      for Android; `$BITRISE_IPA_PATH` for iOS.'
+    title: Path to the app file to be protected
+- jar_file: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Path to DexProtector JAR file. This must be in the same repository as
+      your project.
+    title: Path to DexProtector JAR file
+- config_file: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Path to DexProtector XML configuration file. This must be in the same
+      repository as your project.
+    title: Path to DexProtector XML configuration file
+- license_base64: null
+  opts:
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: DexProtector license file encoded to Base64. To generate the Base64 string
+      of the license file, you can use the bash command `base64 [PATH TO YOUR LICENSE
+      FILE]`.
+    title: DexProtector license file Base64
+- app_platform: auto
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Specify your app’s platform. If you select the `auto` option, the application’s
+      file extension will be used to determine the platform automatically.
+    title: Your app platform
+    value_options:
+    - auto
+    - Android
+    - iOS
+- opts:
+    category: Android specific options
+    is_expand: true
+    summary: 'APK signing mode. Note: `-` means that this option will not be passed
+      to DexProtector as a command line argument. Therefore, if you select `-`, the
+      signing mode must be specified separately in the configuration file.'
+    title: APK signing mode
+    value_options:
+    - '-'
+    - debug
+    - release
+    - google
+    - amazon
+    - none
+  sign_mode: '-'
+- certificate: null
+  opts:
+    category: Android specific options
+    is_expand: true
+    summary: Certificate location. Specifies the path to the signing certificate file.
+      This is necessary if you intend to re-sign the app after it has been DexProtected;
+      this option is therefore only for when the sign mode == `none`. This option
+      can also be specified via the configuration file.
+    title: Certificate location
+- opts:
+    category: Android specific options
+    is_expand: true
+    is_sensitive: true
+    summary: 'For `google` or `amazon` sign mode only. Specifies the target certificate''s
```

Comment:
duplicated appostrophe

Reply:
it's escaping of `'` symbol, because all string wrapped to `'...'`